### PR TITLE
Implement Development profile

### LINKED ISSUE #113: Production/development profiles
Once we have a production ready application, we should create profiles for development and production

### PR DESCRIPTION
**Pull request by skamoen**
This PR implements the dev profiles we so desperately need. It inserts testdata when running in dev profile mode (see run configuration, fill in 'dev' under "Active Profile"). It also removes external dependencies such as a mailserver or a valid mollie key to run all the tests. This still needs some tweaking, but that's for #177 and #160.

Will need to update the README before PR is merged. 

Fixes #113

### CODE REVIEW

**Comment by TimvdLippe** [comment]
LGTM. Will wait for README update before merging.

**Comment by skamoen** [comment]
Updated. ping @TimvdLippe

File: README.md [hunk 1]
```diff
@@ -1,22 +1,50 @@
...
 ### Running the application
 IntelliJ has some neat functionality to make running the application very easy. 
 
 #### Set the properties
-The API has a number of properties that need to be set before the applicaiton can run. There's a template in `resources\config\application.properties.sample`. Copy the contents from this file into a `application.properties`. This file is gitignored because it contains private information such as possible database credetials and payment keys. In the near future, this process will be easier. The sample proprties assume a working PostgreSQL installation running in the background. 
+The API has a number of properties that need to be set before the applicaiton can run.
+There's a template in `resources\config\application.properties.sample`.
+Copy the contents from this file into a `application-dev.properties`.
+This file is gitignored because it contains private information such as possible database credetials and payment keys.
+In the near future, this process will be easier.
```
Comment (TimvdLippe) [inline README.md]: This is not true?

Comment (skamoen) [inline README.md]: You're right. This PR is actually to make it easier.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,49 @@
...
 
 ### Running the application
 IntelliJ has some neat functionality to make running the application very easy. 
 
 #### Set the properties
-The API has a number of properties that need to be set before the applicaiton can run. There's a template in `resources\config\application.properties.sample`. Copy the contents from this file into a `application.properties`. This file is gitignored because it contains private information such as possible database credetials and payment keys. In the near future, this process will be easier. The sample proprties assume a working PostgreSQL installation running in the background. 
+The API has a number of properties that need to be set before the applicaiton can run.
+There's a template in `resources\config\application.properties.sample`.
+Copy the contents from this file into a `application-dev.properties`.
+This file is gitignored because it contains private information such as possible database credetials and payment keys.
```
Comment (TimvdLippe) [inline README.md]: credentials

payment keys -> API keys

File: README.md [hunk 3]
```diff
@@ -1,22 +1,49 @@
...
+The sample proprties assume a working PostgreSQL installation running in the background.
 
 #### Spring Boot
-From the Run/Debug Configuration window, add a new Spring Boot configuration. It needs only two settings to run. The *Main class* should be set to **ch.wisv.areafiftylan.Application** and the *Use classpath of mod...* should be set to your project module, which is the only selectable option in nearly all cases. 
+From the Run/Debug Configuration window, add a new Spring Boot configuration.
```
Comment (TimvdLippe) [inline README.md]: Go to the Run/Debug Configuration window via Run -> Edit Configurations.

File: README.md [hunk 4]
```diff
@@ -1,22 +1,49 @@
...
+It needs only two settings to run.
+The *Main class* should be set to **ch.wisv.areafiftylan.Application** and the *Use classpath of mod...* should be set to your project module, which is the only selectable option in nearly all cases.
```
Comment (TimvdLippe) [inline README.md]: I can't seem to find the `Use classpath of mod...`. Please add the same description type as above on how to find the window/setting.

File: README.md [hunk 5]
```diff
@@ -1,22 +1,49 @@
...
 That's it! Select your new Spring Boot configuration and click Run!. This should launch the application on localhost:9000
 
+## Deploying
+If you want to run the API on your server, you probably don't want to run it from the IDE.
+### Building
+To generate a runnable JAR file, run `./gradlew build`.
+This command will run all tests, and create a runnable JAR file in the root folder.
+### Configuration
+The API has a number of properties that need to be set before the applicaiton can run.
+There's a template in `resources\config\application.properties.sample`.
```
Comment (TimvdLippe) [inline README.md]: Separate folders with `/` to remain consistent in this README.

**Comment by TimvdLippe** [comment]
LGTM

**Pull request merged**